src/components/RichTextEditor/hooks/useEditorValueUpdates.ts

```diff
@@ -0,0 +1,26 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useEffect, useRef } from "react";
+import { Editor, Transforms } from "slate";
+import { clearEditor } from "../utils/editor/clear";
+import { parseRawValue } from "../utils/editor/parseRawContent";
+
+/**
+ * Updates the editor value when the initival value changes
```

Comment:
Suggestions is broken on GH for now 🤷 
There is a typo "when the initival value changes"

Reply:
https://github.com/Frontify/arcade/pull/435